conf/application.conf

```diff
@@ -182,6 +182,6 @@ google-analytics {
   token = N/A
   host = auto
 }
-
+contact-frontend.serviceId = "essttp-frontend"
```

Comment:
It's probably correct, but in that file we mainly use this format
```
contact-frontend {
   serviceId = "essttp-frontend"
}
```

and I think it is better to leave a blank line before and after for readability

Reply:
not sure if the quotes are required either

Reply:
I agree